src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionManager.cs

```diff
@@ -478,7 +478,7 @@ private void LoadAndInitialize(DataCollectorSettings dataCollectorSettings, stri
 
             if (!IsUriValid(dataCollectorUri) && !TryGetUriFromFriendlyName(dataCollectorSettings.FriendlyName, out dataCollectorUri))
             {
-                LogWarning(string.Format(CultureInfo.CurrentUICulture, Resources.Resources.UnableToFetchUriString, dataCollectorSettings.FriendlyName));
+                LogWarning(string.Format(CultureInfo.CurrentCulture, Resources.Resources.UnableToFetchUriString, dataCollectorSettings.FriendlyName));
```

Comment:
What is the reason for changing this from ui culture to current culture?

Reply:
The analyzer is complaining about it :) 

[Rule documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1304) says:

> [CultureInfo.CurrentUICulture](https://docs.microsoft.com/dotnet/api/system.globalization.cultureinfo.currentuiculture) is used only to retrieve localized resources by using an instance of the [System.Resources.ResourceManager](https://docs.microsoft.com/dotnet/api/system.resources.resourcemanager) class.